data/data/aws/main.tf

```diff
@@ -40,8 +40,10 @@ module "bootstrap" {
   vpc_security_group_ids   = [module.vpc.master_sg_id]
   volume_kms_key_id        = var.aws_master_root_volume_kms_key_id
   publish_strategy         = var.aws_publish_strategy
+  iam_role_name            = var.aws_master_iam_role_name
```

Comment:
Remove this. This is coming from a commit that, while in master, is not included in this PR. Alternatively, you can rebase all master onto this branch.